requirements.txt

```diff
@@ -5,7 +5,7 @@ Flask-Bcrypt==0.7.1
 Flask-Marshmallow==0.8.0
 Flask-Migrate==2.1.1
 Flask-Script==2.0.5
-Flask-SQLAlchemy==2.2
+Flask-SQLAlchemy==2.3.0
```

Comment:
Its look like it broke the build https://jenkins.notify.tools/job/notifications-api/1811/